index.html

```diff
@@ -0,0 +1,254 @@
+                        </svg>
+                    </a></li>
+                  </ul>
+                  <button class="about-btn" type="submit"><p>Get my resume</p></button>
+                </div>
+
+                <!-- Developer Skills -->
+
+                <div class="skill-list">
+                    <ul>
+                        <li class="p">Languages
+                            <img src="imgs/Union down.svg" alt="counter" class="counter">
+                            <ul class="lang">
+                                <li class="lang-child"><img src="imgs/Ellipse 1.svg" alt="JavaScript">JavaScript</li>
+                                <li class="lang-child"><img src="imgs/Ellipse 2.svg" alt="HTML">HTML</li>
+                                <li class="lang-child"><img src="imgs/Ellipse 3.svg" alt="CSS">CSS</li>
+                            </ul>
+                        </li>
+                        <li class="p">Frameworks <img src="imgs/Union.svg" alt="counter" class="counter"></li>
+                        <li class="p hr">Skills <img src="imgs/Union.svg" alt="counter" class="counter"></li>
+                    </ul>
+                </div>
+            </section>
+        </section>
+        <div class="space"> </div>
+        <footer class="form-group" id="Contact">
+            <div class="form-child">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it.</p>
+             <form action="https://formspree.io/f/mayklawn" method="POST" class="form-input">
+                <div class="input-field">
+                    <input type="text" name="text" id="name" placeholder="Enter your name" maxlength="30" aria-labelledby="name" required>
+                </div>
+
+                <div class="input-field">
+                    <input type="email" name="email" id="email" placeholder="Enter your email" aria-labelledby="email" required>
+                </div>
+                <div class="input-field">
+                    <textarea name="message" id="comments" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                </div>
+                <div class="input-field">
+                    <button class="btn" type="submit" value="send">Get in touch</button>
+                </div>
+                <span class="error"></span>
+             </form>
+            </div>
```

Comment:
- According to the project requirements, please use the `flex` display property to align contents in this section 👍 👍